go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prometheus/prometheus
 
-go 1.23.0
+go 1.25.3
```

Comment:
Did we need to bump this for a specific reason? I think best practice is leaving it at the minimal version we can compile the binary with.